### ISSUE TITLE
Some performance gap when reproducing results on VATEX dataset.

### ISSUE THREAD

**Issue by tiesanguaixia** [dhg-wei/DeCap#10]
Hi, 

Thank you for your great work on DeCap! I’m trying to reproduce the results reported in your paper on the VATEX dataset for a comparison in my research, but I’ve encountered a significant gap in performance.

Here’s what I’ve done so far:
- For each video, I randomly sampled 10 frames and used a mean pooling mechanism on the frame-level features extracted by the CLIP image encoder to obtain a video-level feature.
- Used all 250,060 captions from the training set as memory to generate the _prefix_.
- Completed evaluation code using COCO-based metrics.

After training, I achieved the best validation results at epoch 10 with:
- **BLEU-4**: 19.71  
- **METEOR**: 19.98  
- **ROUGE_L**: 43.46  
- **CIDEr**: 30.72  

On the test set, the performance is:
- **BLEU-4**: 18.82  
- **METEOR**: 19.22  
- **ROUGE_L**: 42.28  
- **CIDEr**: 26.20  

These results are quite far from the paper’s reported metrics:
- **BLEU-4**: 21.3  
- **METEOR**: 20.7  
- **CIDEr**: 43.1  

Would it be possible for you to share some additional details or resources to help bridge this gap? For example:
1. Training hyperparameters specific to VATEX.  
2. The visual features or instructions for feature extraction.  
3. Model weights or any other relevant checkpoint files.

Thank you so much for your time and help!

**Comment by dhg-wei** [comment]
Sry, I lost the original project code.

Maybe some inference prompts could help improve the performance on VATEX.

**Comment by tiesanguaixia** [comment]
> Sry, I lost the original project code.
> 
> Maybe some inference prompts could help improve the performance on VATEX.

Thank you for your reply, and what are the prompts like?
